src/main.rs

```diff
@@ -276,6 +278,69 @@ fn main() {
         }
     };
 
+    if args.edit_page && args.edit_patch {
+        print_error(
+            enable_styles,
+            &anyhow!("Don't edit-patch and edit-page at the same time"),
+        );
+        process::exit(1);
+    }
+
+    if args.edit_patch || args.edit_page {
+        if args.command.is_empty() {
+            if args.edit_patch {
+                print_error(
+                    enable_styles,
+                    &anyhow!(
+                        "
+No command given
+
+Sample usage :
+    tldr --edit-patch <cmd>",
+                    ),
+                );
+            }
+            if args.edit_page {
+                print_error(
+                    enable_styles,
+                    &anyhow!(
+                        "
+No command given
+
+Sample usage :
+    tldr --edit-page <cmd>",
+                    ),
+                );
+            }
+            process::exit(1);
+        }
+        let command = args.command.join("-").to_lowercase();
+        let custom_page_dir = config
+            .directories
+            .custom_pages_dir
+            .as_ref()
+            .map(PathWithSource::path);
+        if let Some(custom_page_dir) = custom_page_dir {
+            let _ = create_dir_all(custom_page_dir);
+
+            let file_name = if args.edit_patch {
+                format!("{command}.patch.md")
+            } else {
+                format!("{command}.page.md")
+            };
+            let custom_page_path = custom_page_dir.join(file_name);
+            println!("Editing {custom_page_path:?}");
+            let editor = env::var("EDITOR").context("env `EDITOR` not set").unwrap();
+            let _ = std::process::Command::new(editor)
+                .arg(custom_page_path.to_str().unwrap())
+                .status();
+            return;
+        } else {
+            print_error(enable_styles, &anyhow!("Fail to get custom page dir"));
+            process::exit(1);
+        }
```

Comment:
I think this would be nicer if the "else" wouldn't be so far away. How do you feel about refactoring it to be
```rust
let Some(custom_page_dir) = custom_page_dir else {
    // fail
};
```